modules/tide_content_collection/src/Plugin/Field/FieldWidget/ContentCollectionConfigurationWidget.php

```diff
@@ -46,14 +49,27 @@ class ContentCollectionConfigurationWidget extends StringTextareaWidget implemen
    */
   protected $index;
 
+  /**
+   * The Entity Type Manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Specifies whether the field supports both internal and external URLs.
+   */
+  const LINK_GENERIC = 0x11;
```

Comment:
@joshua-salsadigital no need to define your own constant. Just use `LinkItemInterface::LINK_GENERIC`

Reply:
Included and used LinkItemInterface::LINK_GENERIC